encrypted-media-respec.html

```diff
@@ -2331,15 +2365,16 @@
 
         <section id="queue-waitingforkey">
           <h4>Queue a "waitingforkey" Event</h4>
-          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt the current block.
+          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt an encrypted block.
             It should only be called when the <a def-id="htmlmediaelement"></a> object is <a def-id="potentially-playing"></a> and its <a def-id="readystate"></a> is equal to <a def-id="have-future-data"></a> or greater.  
```

Comment:
On Tue, Feb 2, 2016 at 3:42 PM, ddorwin <notifications@github.com> wrote:

> In encrypted-media-respec.html
> <https://github.com/w3c/encrypted-media/pull/139#discussion_r51656285>:
>
> > @@ -2331,15 +2365,16 @@
> >
> >          <section id="queue-waitingforkey">
> >            <h4>Queue a "waitingforkey" Event</h4>
> > -          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt the current block.
> > +          <p>The Queue a "waitingforkey" Event algorithm is run when the CDM cannot decrypt an encrypted block.
> >              It should only be called when the <a def-id="htmlmediaelement"></a> object is <a def-id="potentially-playing"></a> and its <a def-id="readystate"></a> is equal to <a def-id="have-future-data"></a> or greater.
>
> I believe the idea was that it doesn't make sense to be waiting for a key
> if you don't have the data necessary to advance. The ready states are
> strictly about buffering. This was discussed in #100
> <https://github.com/w3c/encrypted-media/issues/100> before this specific
> question was migrated to #129
> <https://github.com/w3c/encrypted-media/issues/129>.
>
​Do you mean to say that "... cannot decrypt the current block." is the
correct language ? I thought I had merged that change correctly from the
main branch. Let me check.​




> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w3c/encrypted-media/pull/139/files#r51656285>.
>